### PR TITLE
first big push

### PR DESCRIPTION
**Pull request by dhawton**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,28 +4,46 @@
...
     "eslint": "6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
     "nodemon": "^2.0.4",
-    "prettier": "^2.0.5"
+    "prettier": "^2.0.5",
+    "prettier-eslint": "^10.1.1",
```
Comment (rahulp959) [inline package.json:46]: What this?

Comment (rahulp959) [reply package.json:46]: Can you not just have it running from within ESLint, as you seem to be attempting to do using https://prettier.io/docs/en/integrating-with-linters.html#disable-formatting-rules

File: src/db/index.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
-const Sequelize = require('sequelize');
-const users = require('./models/User');
-const fs = require('fs');
-const path = require('path');
+const Sequelize = require("sequelize");
```
Comment (rahulp959) [inline src/db/index.js:1]: Let's make sure we get to changing these soon

File: package.json (hunk 1 above)
Comment (dhawton) [reply package.json:46]: It has both, built-in formatting with husky, as well as with linting and it provides tools to VSCode

File: src/db/index.js (hunk 2 above)
Comment (dhawton) [reply src/db/index.js]: Done, commit 8629868.. missed this dir

File: src/db/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      db[modelName].associate(db);
+    }
+  });
+
+  db.sequelize = sequelize;
+
+  return db.sequelize.authenticate();
+};
+
+module.exports = db;
```
Comment (rahulp959) [inline src/db/index.js:45]: Export default?

File: src/routes/v1/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const { Router } = require("express");
+const logbookRouter = require("./logbook");
+
+const router = Router();
+router.use("/logbook", logbookRouter);
+
+module.exports = router;
```
Comment (rahulp959) [inline src/routes/v1/index.js:7]: export and import.

File: src/routes/v1/logbook.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    return next();
+  },
+  startLogbook
+);
+router.get("/:logbookId", getLogbook);
+router.patch("/:logbookId", patchLogbook);
+
+module.exports = router;
```
Comment (rahulp959) [inline src/routes/v1/logbook.js:27]: export

File: src/routes/v1/public.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+const router = Router();
+router.get("/ping", (req, res) =>
+  res.status(200).json({
+    status: "OK",
+    message: "PONG",
+  })
+);
+
+module.exports = router;
```
Comment (rahulp959) [inline src/routes/v1/public.js:11]: export

**Review by rahulp959 — APPROVED** [review]

**Pull request merged**